packages/builder/src/actions/projects.ts

```diff
@@ -287,14 +287,6 @@ export const fetchProjectApplications =
         chainIds,
       });
 
-      applications?.forEach(async (application, index) => {
-        const programName = await dataLayer.getProgramName({
-          projectId: application.round.roundMetadata.programContractAddress,
-        });
-
-        applications[index].round.name = programName || "";
-      });
```

Comment:
^ If we do this, we might have to figure out how to get the round program name for applications

Reply:
Okie this can be removed. Makes sense. Doesn't seem to be used anywhere